src/Inquirer/Validators/PercentageValidator.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+
+namespace InquirerCore.Validators 
+{
+    public class PercentageValidator : IValidator
+    {
+        public bool Validate(string value)
+        {
+            try
+            {
+                var numberWithoutPercent = value.Replace("%", "");
```

Comment:
Reading the code I think that values like "75%%", "7%5" or "%75" will be valid, could you check this? 
Those values that I mentioned could be added to the tests data.